src/mbgl/style/layers/custom_layer.cpp

```diff
@@ -2,7 +2,9 @@
 #include <mbgl/style/layers/custom_layer_impl.hpp>
 #include <mbgl/style/layer_observer.hpp>
 
-#include <mbgl/renderer/layers/render_custom_layer.hpp>
+#ifdef MBGL_RENDER_BACKEND_OPENGL
```

Comment:
this file and probably custom_layer_impl.hpp can move to gl/ so this ifdef is not needed

Reply:
done